functional-tests/bdd-tests/TestBaseNavigation.cs

```diff
@@ -88,17 +88,10 @@ public void ClickLicencesTab()
         {
             // check for failure to load Licences & Authorizations tab successfully
             for (int i = 0; i < 3; i++)
-            {
+            { 
                 ngDriver.Navigate().GoToUrl($"{baseUri}licences");
 
-                try
-                {
-                   if (ngDriver.FindElement(By.XPath("//body[contains(.,'Details')]")).Displayed)      
-                    { 
-                        break; 
-                    }           
-                }
-                catch (Exception)
+                if ((ngDriver.FindElement(By.XPath("//body[contains(.,'Details')]")).Displayed) == false)
                 {
                     ngDriver.Navigate().Refresh();
                     System.Threading.Thread.Sleep(2000);
```

Comment:
same comment as above, please add a path here that will break out of the for loop if everything is fine.